conferences/abstracts/LIBER-submission.md

```diff
@@ -0,0 +1,25 @@
+# Is this the way to reproducibility?
+
+## Submission Details
+Call for papers/Conference website: https://liberconference.eu/liber-2019-call-for-papers/  
+ID: 189  
+Paper (standard presentation)  
+Topics: Open Science, including Open Access  
+Keywords: Research Data Management, Open Science, Reproducibility, Data Librarianship  
+Submitted on 14/01/2019
+
+## Authors
+The Turing Way 3, Patricia Herterich 1 (presenting author), Rosie Higman 2 (presenting author)   
+1 University of Birmingham, United Kingdom;  
+2 University of Manchester, United Kingdom;  
+3 The Alan Turing Institute, United Kingdom;  
+
+## Abstract
+
+Reproducible research is necessary to ensure that scientific work can be trusted. In addition to making publications available open access, funders and publishers are beginning to require that publications include access to the underlying data and the analysis code. The goal is to ensure that all results can be independently verified and built upon in future work, make research transparent for the benefit of the wider society and regain trust in research results.  
```

Comment:
Please reformat to have just one sentence on each line. This is new in the contributing guidelines!! Not something you were supposed to know already!